ddtrace/internal/datadog/profiling/stack_v2/echion/echion/tasks.h

```
@@ -276,10 +285,155 @@ TaskInfo::current(PyObject* loop)
     return TaskInfo::create(reinterpret_cast<TaskObj*>(task));
 }
 
+// ----------------------------------------------------------------------------
+#if PY_VERSION_HEX >= 0x030e0000
+// Python 3.14+: Get tasks from a single thread's linked-list
+inline void
+get_tasks_from_linked_list(uintptr_t head_addr, PyObject* loop, std::vector<TaskInfo::Ptr>& tasks)
+{
+    if (head_addr == 0 || loop == nullptr) {
+        return;
+    }
+
+    // Copy head node struct from remote memory to local memory
+    struct llist_node head_node_local;
+    if (copy_type(reinterpret_cast<void*>(head_addr), head_node_local)) {
+        return;
```

Shouldn't we return an error in this case?